### PR TITLE
Updated Security Section

### PR DESCRIPTION
**Pull request by 0xJayPi**
### Description
Added reference to security reviews of the codebase

### Changes
- Modified `README.md`
- Added `audits/AUDITS.md`

### Related Issue
Related to https://github.com/PaulRBerg/prb-math/issues/197

### CODE REVIEW

File: audits/AUDITS.md
Comment (PaulRBerg) [inline audits/AUDITS.md:1]: The standard place to place security-related information is the `SECURITY.md` file. Let's move this file over there.

**Review by PaulRBerg — CHANGES_REQUESTED** [review]
Hi @0xJayPi, thanks for your PR. I have reviewed it now and I want to make some changes, but it looks like I am not allowed to do that:

> ! [rejected]        main -> main (non-fast-forward)

Could you please give me permission to push to your branch?

Alternatively, please make your changes on a new branch (not on `main`). That will make it work.

**Comment by 0xJayPi** [comment]
Apologies, first time I don't create a branch for simplicity. I'll make the needed changes next week (on vacations leave right now)

**Comment by 0xJayPi** [comment]
@PaulRBerg. I just sent you an invite to my repo. If still needed, I can just move forward and do a new branch and move everything there. Please let me know

**Comment by PaulRBerg** [comment]
@0xJayPi I would prefer not to be added as a collaborator to your repo. Please just create a new PR with a different branch. Thank you.

**Pull request closed without merging**